internal/translation/ipaccesslist/conversion.go

```diff
@@ -0,0 +1,192 @@
+package ipaccesslist
+
+import (
+	"fmt"
+	"net"
+	"strings"
+	"time"
+
+	"go.mongodb.org/atlas-sdk/v20231115008/admin"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/timeutil"
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1/project"
+)
+
+type IPAccessEntry struct {
+	CIDR             string
+	AWSSecurityGroup string
+	DeleteAfterDate  *time.Time
+	Comment          string
+}
+
+func (i *IPAccessEntry) ID() string {
+	if i.CIDR != "" {
+		return i.CIDR
+	}
+
+	return i.AWSSecurityGroup
+}
+
+func (i *IPAccessEntry) IsExpired() bool {
+	if i.DeleteAfterDate == nil {
+		return false
+	}
+
+	return i.DeleteAfterDate.Before(time.Now())
+}
+
+type IPAccessEntries map[string]*IPAccessEntry
+
+func (i IPAccessEntries) GetActives() IPAccessEntries {
+	entries := make(IPAccessEntries, len(i))
+	for ix, entry := range i {
+		if !entry.IsExpired() {
+			entries[ix] = entry
+		}
+	}
+
+	return entries
+}
+
+func (i IPAccessEntries) GetExpired() IPAccessEntries {
+	entries := make(IPAccessEntries, len(i))
+	for ix, entry := range i {
+		if entry.IsExpired() {
+			entries[ix] = entry
+		}
+	}
+
+	return entries
+}
+
+func ToAKO(ipAccessEntries IPAccessEntries) []project.IPAccessList {
```

Comment:
I believe the convention elsewhere was `fromAtlas` and `toAtlas`. Let's stick to a common scheme.

Reply:
`func FromInternal(ipAccessEntries IPAccessEntries) []project.IPAccessList {`

Reply:
@helderjs do you mind to add the "source signifier" convention into the translation layer proposal, including the rationale (aka the "why").

Reply:
Yes, I thought we would only use 'toAtlas` and `fromAtlas` everywhere.

We also intend to overuse the AKO acronym in our code a lot.